ansible/idr-searchengine.yml

```diff
@@ -151,6 +161,35 @@
       volumes:
         - "{{ apps_folder }}/searchengine/searchengine/:/etc/searchengine/"
 
+# automatic refreshing  when configuration change is detected
+
+  - name: Set automatic refreshing the configuration
+    become: yes
+    docker_container:
+      image: "{{ searchengine_docker_image }}"
+      name: elastics_password
+      cleanup: True
+      #auto_remove: yes
+      command: "set_automatic_refresh -a {{ automatic_refresh }}"
+      state: started
+      volumes:
+        - "{{ apps_folder }}/searchengine/searchengine/:/etc/searchengine/"
+
+# restore the elastic indexed data from the backup
+  # This required that the  indexed data snapshot be copied to the
+  # backup folder before running the  elasticsearch deployment playbook
```

Comment:
For the OMERO database, binary repository and NGINX cache, the volumes are being cloned from the former production deployment and used during the provisioning step.
Is it possible to follow the same strategy here?

Reply:
We need to transfer the snapshot backup data from the current production server to the next production server and restore it on the new machine. Would it be possible to mount a small 20 GB volume for the backup data, which we could then clone for deployment?

Reply:
A dedicated volume would certainly be consistent with the pattern established for cloning the database and the OMERO binary repository from the previous production environment to a new production or a pilot.

https://github.com/IDR/deployment/blob/f7e8bb9699a64cb291c8dde09f519bb7c0eaf756/ansible/openstack-create-publicidr.yml#L151-L171 will need to be adjusted to declare a new volume and https://github.com/IDR/deployment/blob/master/scripts/os-idr-snapshot.sh will also need to be updated to snapshot this new volume.